pyproject.toml

```diff
@@ -234,40 +234,39 @@ extend-exclude = [
 ]
 
 [tool.ruff.lint]
+extend-safe-fixes = [
+  "TID252", # absolute imports
+]
 extend-select = [
-  "F",    # Pyflakes
-  "E",    # pycodestyle errors
-  "W",    # pycodestyle warnings
-  "I",    # isort
-  "UP",   # pyupgrade
-  "B",    # flake8-bugbear
-  "C4",   # flake8-comprehensions
-  "PIE",  # flake8-pie
-  "TID",  # flake8-tidy-imports (absolute imports)
-  "PGH",  # pygrep-hooks
+  "F", # Pyflakes
```

Comment:
Doesn't black usually do 2 whitespaces for comments?

Reply:
Oh, good question.
I shouldn't have even formatted this even, but pre-commit did that ;)

Reply:
which hook was that? As far as I can tell, we don't have anything that would format TOML (yet, we can totally change that).

Reply:
Oh, might have been format on save in vscode.
I will have to check that.

Reply:
> we don't have anything that would format TOML (yet, we can totally change that).

Stumbled across this and thought I'd make https://github.com/pydata/xarray/pull/9840 :)